### PR TITLE
*: update CI go version and fix test

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @andelf @disksing @siddontang

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 language: go
 
 go:
-  - 1.7.3
```
Comment (siddontang) [inline .travis.yml]: can we support go 1.7 too?

Comment (andelf) [inline .travis.yml]: this is a yaml list, just list all versions.

Comment (siddontang) [inline .travis.yml]: I know, what I mean is to test with go 1.7 too, not only 1.8.

**Comment by siddontang** [comment]
CI failed

File: pkg/etcdutil/etcdutil.go [hunk 2]
```diff
@@ -68,6 +68,7 @@ func CheckClusterID(localClusterID types.ID, um types.URLsMap) error {
 		// For tests, change scheme to http.
 		// etcdserver/api/v3rpc does not recognize unix protocol.
 		if u.Scheme == "unix" || u.Scheme == "unixs" {
+			//directly use http://localhost
```
Comment (siddontang) [inline pkg/etcdutil/etcdutil.go]: `// directly`

File: .travis.yml (hunk 1 above)
Comment (siddontang) [inline .travis.yml]: please support go 1.7

Comment (nolouch) [inline .travis.yml]: I update go version to 1.8 in travis ci and 1.7 in circle ci. same as tidb.

**Comment by disksing** [comment]
LGTM

**Pull request merged**